device-types/Cisco/ISR-1127-8PLTEP.yaml

```diff
@@ -0,0 +1,43 @@
+---
+manufacturer: Cisco
+model: ISR 1127-8PLTEP
+part_number: C1127-8PLTEP
+slug: cisco-isr-1127-8pltep
+weight: 4.53
+weight_unit: lb
+u_height: 1
```

Comment:
I do not believe this is a racked device. It looks like by default this is a desktop/shelf device. If this is true please modify this to `u_height: 0`